c2corg_ui/templates/waypoint/edit.html

```diff
@@ -90,7 +90,7 @@ <h2 class="heading show-phone"><span translate>title</span></h2>
                  ng-class="{ 'has-error': editForm.lang.$touched && editForm.lang.$invalid, 'has-success': editForm.lang.$valid}">
               <label><span translate>lang</span> *</label>
               % if waypoint_lang:
-              <input class="form-control" disabled x-translate value="${waypoint_lang}">
+              <input class="form-control" disabled value="{{mainCtrl.translate('${waypoint_lang}')}}">
```

Comment:
You mean as a filter like ``value="{{'${waypoint_lang} | translate}}'"``?
In that case ``${waypoint_lang}`` is pushed to the POT files by the extractor.
Using ``x-translate`` as a filter does not work (no effect).

If as an attribute (as the original ``x-translate``), the translation does not work because the directive does not apply to the ``value`` content.